src/httpserver.cpp

```diff
@@ -175,7 +176,7 @@ static bool InitHTTPAllowList()
         LookupSubNet(strAllow.c_str(), subnet);
         if (!subnet.IsValid()) {
             uiInterface.ThreadSafeMessageBox(
-                strprintf("Invalid -rpcallowip subnet specification: %s. Valid are a single IP (e.g. 1.2.3.4), a network/netmask (e.g. 1.2.3.4/255.255.255.0) or a network/CIDR (e.g. 1.2.3.4/24).", strAllow),
+                strprintf(_("Invalid -rpcallowip subnet specification: %s. Valid are a single IP (e.g. 1.2.3.4), a network/netmask (e.g. 1.2.3.4/255.255.255.0) or a network/CIDR (e.g. 1.2.3.4/24)."), strAllow),
```

Comment:
nit: make this `NoTranslate`. All relevant documentation is in English anyway.

Reply:
Agree. This is a terrible message to translate.

Reply:
Done.

Reply:
This has not yet been addressed

Reply:
I'd say we should leave all new translations for a later pull. You can blindly mark everything `Untranslated` for now

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/16224#issuecomment-623798715).